src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -234,10 +234,11 @@ public async Task ShutdownAsync()
 
         public async Task InvokeAsync(ScriptInvocationContext invocationContext)
         {
+            // This could throw if no initialized workers are found. Shut down instance and retry.
+            IEnumerable<IRpcWorkerChannel> workerChannels = await GetInitializedWorkerChannelsAsync();
+            var rpcWorkerChannel = _functionDispatcherLoadBalancer.GetLanguageWorkerChannel(workerChannels, _maxProcessCount);
```

Comment:
Remove try...catch as the function not loaded error is also not user error.

Reply:
done!